product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Security and Compliance"
+---
+
+### Security Governance
+
+- Cloud Security Compliance officer is responsible for maintaining compliance targets for their entire organization.
+- EDB Cloud infrastructure periodically undergoes SOC2 audit trials to ensure that EDB Cloud is always compliant with security requirements.
+- All EDB staff undergo background checks before they are hired.
+- All EDB staff are required to take mandatory security training.
+- The EDB Software Development Life Cycle integrates continuous security scans of added/updated code.
+
+
+### EDB Cloud Security Features
+EDB Cloud is a Database as a Service (DBaaS) which runs in your own Cloud account, isolating your data from other users & giving you control over our access to it. The key security features of EDB Cloud are as follows:
+- **Segregation of IT environments:** Customers database clusters are installed and managed in their own cloud environment. Complete segregation of customer's data is ensured.
```

Comment:
As noted above, I think this warrants a dedicated section, maybe along with Shared Responsibility.

Reply:
There is shared responsibility section: https://docs.google.com/spreadsheets/d/1PhKLCL9DMcvcZJVSWNMe42LUUa4WaB2-fLPFJose3O8/edit#gid=948888424

Reply:
The Shared Responsibility section is here https://60f76388eb1d304637d07e7b--edb-docs-staging.netlify.app/docs/edbcloud/latest/overview/04_responsibility_model/
it is compilation of your Security RACI doc https://60f76388eb1d304637d07e7b--edb-docs-staging.netlify.app/docs/edbcloud/latest/overview/04_responsibility_model/

Reply:
Yeah, that's pretty good - after reviewing the whole thing here I think it's this bit that needs some words:

> We should have some words here about not touching the Subscriptions we're working in, eh?